src/frontend/js/types/index.ts

```diff
@@ -25,7 +25,7 @@ export interface Referral {
 
 export interface ReferralLite
   extends Pick<Referral, 'assignees' | 'due_date' | 'id' | 'object' | 'state'> {
-  users: UserLite[];
+  users: User[];
```

Comment:
This change seems wrong to me: referral lites are still including only user lites